doc/contributing.md

```diff
@@ -1,16 +1,171 @@
 # Contributing Guide
 
-This is the contrituting guide to HoloViz, and the reference to better understand how the HoloViz project is managed.
+This guide includes general guidelines on how to contribute to HoloViz. Read the [Operating Guide](TODO) if you are interested to learn how an Open-Source project like HoloViz operates.
 
-## TL;DR
+## Why contribute?
 
+There are many reasons why you would contribute to the HoloViz project, including:
 
+* You appreciate the HoloViz project, its mission and values, and would like to help
```

Comment:
should we add a link to 'mission and values'... maybe the `background` or `about` page?